### ISSUE TITLE
Clear selected photo from imageRow - code written

### ISSUE THREAD

**Issue by btanner** [xmartlabs/Eureka#228]
I was trying to do the right thing and figure out how to do a pull request, but learning iOS, Git, GitHub, XCode, and Eureka all in 2 days seems to have taxed me enough that I can't figure out how to do it.

I am planning to write quite a bit of additional custom code to extend Eureka.  I ended up implementing a photo clear option by modifying your code instead of adding a new custom row to mine, and I thought the code might just be generally useful.

Insert the following around line 717 of the current Rows.swift (right before the camera, library,etc options)

    //if the photo is set, give the option to clear it
        if !(value == nil) {
            let clearPhotoOption = UIAlertAction(title: "Clear Photo", style: .Default, handler: { [weak self] (alert: UIAlertAction) -> Void in
                self?.value=nil;
                self?.customUpdateCell();
                })
            sourceActionSheet.addAction(clearPhotoOption)
        }

**Comment by mtnbarreto** [comment]
@btanner Could you please make a pull request with this?

```swift
if let _ = value {
        let clearPhotoOption = UIAlertAction(title: "Clear Photo", style: .Default, handler: { [weak self] (alert: UIAlertAction) -> Void in
            self?.value=nil;
            self?.updateCell();
            })
        sourceActionSheet.addAction(clearPhotoOption)
    }
```

I would also like to add a new property to `ImageRow` to determine if "Clear Photo" should be shown. 

Regards

**Comment by mats-claassen** [comment]
Merged in #255
